### PR TITLE
Debug/validating registration

### PR DESCRIPTION
**Pull request by YucenLyc**

### CODE REVIEW

File: server/routes/auth-routes.js [hunk 1]
```diff
@@ -9,9 +9,14 @@ const router = express.Router();
 router.post('/login', async (req, res) => {
   try {
     const {email, password} = req.body;
+    console.log("this is the login route, it's hit")
     const users = await pool.query('SELECT * FROM users WHERE users.email = $1', [email]);
+
     if (users.rows.length === 0) return res.status(401).json({error: "email is incorrect"});
+    
     // password check:
+    console.log("password:", password);
```
Comment (christophercpang) [inline server/routes/auth-routes.js:18]: Remove console logs if not required

File: server/routes/users-routes.js [hunk 2]
```diff
@@ -2,27 +2,40 @@ import express from 'express';
...
-    res.json({users : users.rows})
-  } catch (error){
-    res.status(500).json({error:error.message})
+    res.json({ users: users.rows })
+  } catch (error) {
+    res.status(500).json({ error: error.message })
   }
 });
 
+router.post('/testing', (req, res) => {
```
Comment (christophercpang) [inline server/routes/users-routes.js:22]: Do we still need this testing route?

File: server/routes/users-routes.js [hunk 3]
```diff
@@ -2,27 +2,40 @@ import express from 'express';
...
+      res.json({ error: "Invalid User Input" })
+    }
+    res.json({message: 'it hit the registration route'})
     const newUser = await pool.query('INSERT INTO users(name, email, password) VALUES ($1, $2, $3) RETURNING *', [req.body.name, req.body.email, hashedPassword]);
-    console.log("New User Created!")
-    res.json({users: newUser.rows[0]})
-  } catch (error) {
-    res.status(500).json({error: error.message});
-  }
+    console.log(newUser)
```
Comment (christophercpang) [inline server/routes/users-routes.js:36]: Remove console logs

**Review by christophercpang — CHANGES_REQUESTED** [review]
Couple of comments - let me know if you have any questions!

File: client/frontend-releasify/src/Pages/Register.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  const baseURL = process.env.REACT_APP_API_BASE_URL;
+  console.log("this is the base URL:", baseURL);
+
+  const register = (event) => {
+    if (password === confirmPassword) {
+      console.log(baseURL)
+      axios.post( baseURL + '/api/users/register', {
+        name: name,
+        email: email,
+        password: password,
```
[the remark this answers is not in the document]
Comment (YucenLyc) [reply client/frontend-releasify/src/Pages/Register.js:19]: good to know!! ty!

File: server/routes/users-routes.js (hunk 2 above)
Comment (YucenLyc) [reply server/routes/users-routes.js:22]: Actually, no, will remove it!

File: client/frontend-releasify/src/Pages/Login.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Redirect } from "react-router-dom";
+import '../Styles/Login.css'
+import { baseURL } from "../Service/BaseURLService"
+
+
+export default function Login() {
```
Comment (christophercpang) [inline client/frontend-releasify/src/Pages/Login.js:8]: I think we should keep our code consistent in terms of how we define functions, so let's change this to

```
const Login = () => {
...
}

export default Login
```

**Review by christophercpang — APPROVED** [review]
Looks good!

**Comment by YucenLyc** [comment]
fix registration FE error; clean up code;

**Pull request merged**